doc/source/user/absolute_beginners.rst

```diff
@@ -1251,86 +1295,85 @@ function that can help you access this information. This means that nearly any
 time you need more information, you can use ``help()`` to quickly find the
 information that you need.
 
-For example, ::
+For example::
 
   >>> help(max)
-
-Will return::
-
   Help on built-in function max in module builtins:
+  <BLANKLINE>
+  max(...)
+      max(iterable, *[, default=obj, key=func]) -> value
+      max(arg1, arg2, *args, *[, key=func]) -> value
+  <BLANKLINE>
+      With a single iterable argument, return its biggest item. The
+      default keyword-only argument specifies an object to return if
+      the provided iterable is empty.
+      With two or more arguments, return the largest argument.
+  <BLANKLINE>
 
-  max(...) max(iterable, *[, default=obj, key=func]) -> value max(arg1, arg2,
-  *args, *[, key=func]) -> value
-
-      With a single iterable argument, return its biggest item. The default
-      keyword-only argument specifies an object to return if the provided
-      iterable is empty. With two or more arguments, return the largest
-      argument.
 
 Because access to additional information is so useful, IPython uses the ``?``
 character as a shorthand for accessing this documentation along with other
 relevant information. IPython is a command shell for interactive computing in
 multiple languages.
 `You can find more information about IPython here <https://ipython.org/>`_.
 
-For example, ::
+For example:
 
-  >>> max?
+.. code-block:: ipython
 
-Will return::
+  In [0]: max?
+  
+  Out[0]: max(iterable, *[, default=obj, key=func]) -> value
```

Comment:
Does this actually appear in `Out[]`?

Reply:
if I change something there and rerun the refguide-check, it does not discover the discrepancy. However when running `python runtests.py --ipython` then at the prompt typing `max?`, that is the output. Maybe we could enhance refguide-check to do more, but I think we would be better off to move toward astropy's use of [`pytest-doctestplus`](https://pypi.org/project/pytest-doctestplus/) instead

Reply:
Not on my machine it isn't:
```
In [14]: max?
Docstring:
max(iterable, *[, default=obj, key=func]) -> value
max(arg1, arg2, *args, *[, key=func]) -> value

With a single iterable argument, return its biggest item. The
default keyword-only argument specifies an object to return if
the provided iterable is empty.
With two or more arguments, return the largest argument.
Type:      builtin_function_or_method
```
Note the lack of `Out[ ]` there

Reply:
When using `?` or `??` (here and below), ipython doesn't print `Out[n]`, and the output is aligned with the left edge of the terminal.  E.g.

```
In [1]: max?                                                                                                                
Docstring:
max(iterable, *[, default=obj, key=func]) -> value
max(arg1, arg2, *args, *[, key=func]) -> value

With a single iterable argument, return its biggest item. The
default keyword-only argument specifies an object to return if
the provided iterable is empty.
With two or more arguments, return the largest argument.
Type:      builtin_function_or_method
```

and
```
In [3]: double??                                                                                                            
Signature: double(a)
Source:   
def double(a):
    '''Return a * 2'''
    return a * 2
File:      ~/<ipython-input-2-8e354c1d5335>
Type:      function
```
If we have to keep your changes as is to prevent CI errors, then we should also have a github issue about the reguide check code, as this looks like a shortcoming of that code.

Reply:
> if I change something there and rerun the refguide-check, it does not discover the discrepancy. &emdash; @mattip

> If we have to keep your changes as is to prevent CI errors &emdash; @WarrenWeckesser 

Warren, it sounds like CI doesn't care about this at all, not that it forces us to do the wrong thing.

Reply:
@WarrenWeckesser can you push a fix to my branch with the correct formatting? I don't really use ipython so I am not sure how this should look.

Reply:
@mattip, here's a commit with my suggested change: https://github.com/WarrenWeckesser/numpy/commit/5b252f2f51d850217abeae92e966a487dd6be839

Reply:
I pushed that change to this PR.